src/api/opportunity/content-types/opportunity/schema.json

```diff
@@ -84,6 +73,11 @@
       ],
       "required": true,
       "default": "Front-End Developer"
+    },
+    "roleCategory": {
+      "type": "component",
+      "repeatable": false,
+      "component": "role-category.role-category"
```

Comment:
What this repeatable means ?

Reply:
Repeatable means make this column a list. That would make roleCategory a list of values which wouldn't make sense, because each opportunity belongs to one category.

Reply:
OK, thank you, it is clear now.